vpr/src/route/route.cpp

```diff
@@ -441,8 +441,8 @@ bool route(const Netlist<>& net_list,
         } else {
             pres_fac *= router_opts.pres_fac_mult;
 
-            /* Avoid overflow for high iteration counts, even if acc_cost is big */
-            pres_fac = update_draw_pres_fac(std::min(pres_fac, static_cast<float>(HUGE_POSITIVE_FLOAT / 1e5)));
+            /* Set the maximum pres_fac to the value passed by the command line argument */
+            pres_fac = update_draw_pres_fac(std::min(pres_fac, router_opts.max_pres_fac));
```

Comment:
Let's rename this function to something a bit more descriptive. update_pres_fac_and_store_for_draw ().
Or move pres_fac to the routing context if it's cleaner.